recipes/_linux.rb

```diff
@@ -23,9 +23,13 @@
 when "debian"
   include_recipe "apt"
 
+  package 'apt-transport-https' do
+    action :remove
```

Comment:
we need to install this not remove it.

Reply:
Sorry - this was a test commit, just trying to validate an assumption on the Travis build.

Reply:
ok, no problem if it is a wip then it's best to label the PR title with `[WIP]`

Reply:
Also sadly we never added travis and kitchen integration. It's something I want to add when I have the time.